src/test/integration/services/persisters/s3Persister.test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const chai = require('chai');
+chai.should();
+
+describe('[INTEGRATION] s3Persister', function() {
+    const IntegrationTest = require('../../integrationTest');
+
+    IntegrationTest.Setup({
+        database: {
+        s3: {
```

Comment:
`IntegrationTest.Setup()` is a method that instantiates the RootService, and thus instantiates all of the other services. The object served to `Setup()` is the entirety of the key-values that will be thrown into `ConfigService` -- allowing for something like `IntegrationTest.Root.config['database']['s3']['host']` to return "localhost"

Now, with all that said, there's no point in this current config setup for the following reasons:
1. S3 is not a database, so "s3" does not need to be in the "database" blob.
2. S3 does not have any necessary config data that needs to be saved. (If you have to ask yourself if it does, the answer is probably no)
3. If it did, that same information should be added to the default `/config/config.json`

Change lines 9-19 to `IntegrationTest.Setup();`